test/integration/IntegrationTest.cpp

```diff
@@ -223,6 +224,16 @@ api::ConferenceEndpointExtendedInfo IntegrationTest::getEndpointExtendedInfo(con
     return api::Parser::parseEndpointExtendedInfo(endpointRequest.getJsonBody());
 }
 
+size_t getNumWorkingThreads()
```

Comment:
This name in the same file creates confusion as there is already a getNumWorkerThreads.

Reply:
Agree.